pkgs/tools/misc/mons/default.nix

```diff
@@ -12,17 +22,46 @@ stdenv.mkDerivation rec {
     fetchSubmodules = true;
   };
 
-  patches = [
-    # Substitute xrandr path with @xrandr@ so we can replace it with
-    # real path in substituteInPlace
-    ./xrandr.patch
-  ];
-
-  postPatch = ''
-    substituteInPlace mons.sh --replace '@xrandr@' '${xrandr}/bin/xrandr'
-  '';
+  solutions = {
+    mons = {
+      scripts = [
+        "bin/mons"
+        "lib/libshlist/liblist.sh"
+      ];
+      interpreter = "${bash}/bin/sh";
```

Comment:
The reason I am using `sh` instead of `bash` here is because `mons` is written with for a POSIX compliant shell, not necessary bash.